lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -195,179 +354,428 @@ def process_state(self):
             self.module.fail_json(msg=to_native(runtime_fault.msg))
         except vmodl.MethodFault as method_fault:
             self.module.fail_json(msg=to_native(method_fault.msg))
-        except Exception as e:
-            self.module.fail_json(msg=to_native(e))
+        # TODO: delete
```

Comment:
Any specific reason to delete this generic exception. In my opinion, it will help when all given exception fails. It will show nice error message to user, rather than showing stacktrace.

Reply:
A generic exception never happens in this case based on the documentation. 

Btw. The exception handling should be slightly changed in all modules. `.msg` is too generic and the same message you can see in the vSphere Client. Most of the time details about the error are included in another property.

Reply:
I agree a generic exception never happens in this case as per documentation but keeping it won't harm. 

Yes, I agree we have to manage exception depending upon exception, but I think that should be done on the case to case basis.

Reply:
> Yes, I agree we have to manage exception msg depending upon exception, but I think that should be done on the case to case basis.

I don't waste my time to test this on a case by case basis... This property just need to be added as an additional parameter. If it's `None`, it's just not included...

Reply:
ok.